### PR TITLE
237 unify trailing blackslashes in urls

### PR DESCRIPTION
**Pull request by dorianjagusch**
All's said

### CODE REVIEW

File: backend/backend/urls.py [hunk 1]
```diff
@@ -71,11 +71,11 @@
...
 	# Tournament views
 	path('tournaments/', TournamentListView.as_view()),
-	path('tournaments/<int:tournament_id>', TournamentDetailView.as_view()),
+	path('tournaments/<int:tournament_id>/', TournamentDetailView.as_view()),
 	path('tournaments/<int:tournament_id>/players/', TournamentPlayerListView.as_view()),
-	path('tournaments/<int:tournament_id>/players/<int:tournamentplayer_id>', TournamentPlayerDetailView.as_view()),
+	path('tournaments/<int:tournament_id>/players/<int:tournamentplayer_id>/', TournamentPlayerDetailView.as_view()),
 	path('tournaments/<int:tournament_id>/matches/', TournamentMatchListView.as_view()),
-	path('tournaments/<int:tournament_id>/matches/<int:tournament_match_id>', TournamentMatchDetailView.as_view()),
+	path('tournaments/<int:tournament_id>/matches/<int:tournament_match_id>/', TournamentMatchDetailView.as_view()),
```
Comment (merituulie) [inline backend/backend/urls.py:78]: wasnt @sakarisalmi already fixing these in one PR?

**Review by azarSarikhani — APPROVED** [review]
Nice!

File: backend/backend/urls.py (hunk 1 above)
Comment (dorianjagusch) [reply backend/backend/urls.py:78]: Shouldn't have, since we said, I'd do it...

Comment (dorianjagusch) [reply backend/backend/urls.py:78]: Alas he has... Ill  close this and add the append backslash later

**Pull request closed without merging**